keyboards/keychron/c3_pro/ansi/red/config.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2023 @ Keychron (https://www.keychron.com)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define DRIVER_COUNT 1
+#define DRIVER_ADDR_1 0b1110100
```

Comment:
@lalalademaxiya1 would you be able to tell me the possible values for the I2C address of the CKLED2001 please? Is it similar to IS31FL3731:
```
00 AD <-> GND
01 AD <-> SCL
10 AD <-> SDA
11 AD <-> VCC
```

Reply:
Yes, it is. The follow picture pick from the official datasheet

Reply:
Thank you. Is this datasheet publicly available? I cannot find it.

Reply:
Hi, download and read it at this link: https://1drv.ms/b/s!AoXHaYfG8GA8zQq7vMATwQIcs4Uh?e=0i8eWT